### PR TITLE
Docker restructuring

### PR DESCRIPTION
**Pull request by nikostsagk**
- builds images with tag `local`
- before the user pushes the image he should tag it with the tag `deploy`
- `detector.launch` launches both server and client
- `detector_server.launch` launches only the server
- `detector_client.launch` launches only the client

### CODE REVIEW

**Comment by nikostsagk** [comment]
This can be safely merged

**Comment by rjwb1** [comment]
is this ok to merge?

**Comment by saulgoldsaga** [comment]
> is this ok to merge?

Can I check on Monday 😉

**Comment by rjwb1** [comment]
I'm merging this

**Comment by rjwb1** [comment]
tested on jetson here with both backends

**Comment by saulgoldsaga** [comment]
@nikostsagk Is there a way to make the docker-compose.yaml location not hardcoded in run_backend.sh?

**Comment by nikostsagk** [comment]
yes! by moving run_backend.sh in the same directory with the docker-compose.yaml

File: scripts/run_backend.sh [hunk 1]
```diff
@@ -10,91 +11,18 @@ fi;
...
-#     ROS_LOCAL_IP="${ROS_IP:-"127.0.0.1"}"
-#     docker-compose -f ../docker/docker-compose.yml run gripper-perception
-# else
-#     echo "No valid container for backend ${image_name} can be started"
-#     exit 1
-# fi
+# Bring container down if already up (avoid corruption)
+docker-compose -f $HOME/catkin_ws/src/rasberry_perception/docker/docker-compose.yml rm -sf $SERVICE_NAME
+# Run backend
+docker-compose -f $HOME/catkin_ws/src/rasberry_perception/docker/docker-compose.yml run $SERVICE_NAME
```
Comment (saulgoldsaga) [inline scripts/run_backend.sh:28]: @nikostsagk Here it is hardcoded

Comment (nikostsagk) [reply scripts/run_backend.sh:28]: Yes, surprisingly in the jetson doesn't work for relative paths, it does for my system though. 

I wouldn't merge yet, it seems there are a lot of differences between amd64/arm64 that I'm not yet aware of...

Comment (rjwb1) [reply scripts/run_backend.sh:28]: Can i move this file to the scripts folder and then commit?

Comment (saulgoldsaga) [reply scripts/run_backend.sh:28]: It's ok with me

Comment (nikostsagk) [reply scripts/run_backend.sh]: run_backend.sh has now been moved

**Pull request merged**